conpot/protocols/modbus/modbus_server.py

```diff
@@ -23,34 +27,60 @@ class ModbusServer(modbus.Server):
     def __init__(self, template, template_directory, args, timeout=5):
 
         self.timeout = timeout
+        self.delay = None
+        self.mode = None
         databank = slave_db.SlaveBase(template)
 
         # Constructor: initializes the server settings
-        modbus.Server.__init__(self, databank if databank else modbus.Databank())
+        modbus.Server.__init__(
+            self, databank if databank else modbus.Databank())
```

Comment:
We are not super strict about it but usually everything below 120 chars is fine ;)